modules/twinklexfd.js

```diff
@@ -1616,6 +1705,27 @@ Twinkle.xfd.callback.evaluate = function(e) {
 			// find target and pass main as the callback
 			Twinkle.xfd.callbacks.rfd.findTarget(params, Twinkle.xfd.callbacks.rfd.main);
 			break;
+
+		case 'rm':
+			params = { reason: reason, newname: newname, rmtr: rmtr };
+			var nomPageName = rmtr ?
+				'Wikipedia:Requested moves/Technical requests' :
+				new mw.Title(Morebits.pageNameNorm).getTalkPage().toText();
```

Comment:
Do we want to be concerned with redirected talk pages?

Reply:
Actually, if the talkpage message follows the redirect, which by all accounts it should, the template would need to use the `current1` parameter, right?  Is there a problem to just *always* setting `current1=Morebits.pageNameNorm` (in `getDiscussionWikitext`) for every use of the template?  AFAICT it looks fine in preview, not sure if the bot gets cranky.

Reply:
As long as the value provided to `|current1=` is same as subject page name, the substiuted template output is same as when `|current1=` is not provided. When the value is not same, it gets treated as a multiple-page nomination (`{{requested move/dated|multiple=yes
|current1=xxx|new1=xxx}}`), and the reason text appears after a line break. Pretty sure the bot won't get cranky, as it must be designed to handle multiple-page nominations.

Reply:
> You mean with a `wikipedia_page.setFollowRedirect(true);`? 

Yes. 0c00748 looks good.